pkg/config/sync.go

```diff
@@ -281,6 +281,11 @@ func logCurrentConfig() {
 			config = append(config, fmt.Sprintf("workspace.ignoredUnrecoverableEvents=%s",
 				strings.Join(Workspace.IgnoredUnrecoverableEvents, ";")))
 		}
+		for storageClassName, PVCSize := range Workspace.StorageClassSize {
+			if PVCSize != defaultConfig.Workspace.StorageClassSize[storageClassName] {
+				config = append(config, fmt.Sprintf("workspace.storageClassSize[%s]=%s", storageClassName, &PVCSize))
```

Comment:
I'm not sure if this is the best format (i.e. with the `[]`) to use for displaying non-default PVC storage sizes, though it is consistent with its internal representation as a map.

Also, maybe this logged information should occur right after `PVCName` is logged, for consistency with the CRD?

Reply:
I think you may be getting thrown off by using a `map[string]resource.Quantity` here -- there's no yaml-level difference in defining `storageClassSize` as a `map[string]resource.Quantity` and something like
```go
storageClassSize struct {
  perWorkspaceSize resource.Quantity `json:"per-workspace,omitempty"`
  commonSize resource.Quantity `json:"common,omitempty"`
}
```
(except that we're defining all the valid keys in the map)

In short, the format I'd expect is 
```suggestion
				config = append(config, fmt.Sprintf("workspace.storageClassSize.%s=%s", storageClassName, &PVCSize))
```

Reply:
Thank you for the clarification :) Your suggestion is what I initially was going to write, but wasn't sure if it was correct